frontend/.vscode/settings.json

```diff
@@ -0,0 +1,120 @@
+{
+	"prettier.documentSelectors": [
+		"**/*.svelte"
+	],
+	"tailwindCSS.classAttributes": [
+		"class",
```

Comment:
Why are you specifying all of those as class attributes? Also, the `.vscode` folder should be excluded from version control. Let me know if you know how to exclude or if you would want me to do it instead.

Reply:
I didn't write this code; I think it was included when I first set up the skeleton project.
I'll see if I can omit the .vscode folder myself. I'll let you know if there's any difficulties.